cartography/intel/aws/ec2/security_groups.py

```diff
@@ -128,7 +128,7 @@ def load_ec2_security_groupinfo(neo4j_session, data, region, current_aws_account
         )
 
         load_ec2_security_group_rule(neo4j_session, group, "IpPermissions", aws_update_tag)
-        load_ec2_security_group_rule(neo4j_session, group, "IpPermissionEgress", aws_update_tag)
+        load_ec2_security_group_rule(neo4j_session, group, "IpPermissionsEgress", aws_update_tag)
```

Comment:
Interesting, this has been a typo all this time? :o

Reply:
At least the typo is there in master now --- I was very surprised too when I realized that there are no egress rules in the db.